sdks/python/apache_beam/io/gcp/bigquery_tools.py

```diff
@@ -527,6 +531,7 @@ def _insert_load_job(
                     schema=job_schema,
                     writeDisposition=write_disposition,
                     createDisposition=create_disposition,
+                    schemaUpdateOptions=schema_update_options,
                     sourceFormat=source_format,
                     useAvroLogicalTypes=True,
                     autodetect=schema == 'SCHEMA_AUTODETECT',
```

Comment:
We should confirm early in WriteToBigQuery that the schema update options are present in either `schema_update_options` or `additional_load_parameters` and resolve if options are present in both. 

If options are present in `additional_load_parameters` at this point, we would have two `schemaUpdateOptions` parameters here.

Reply:
We should also allow backwards compatibility. If users want to keep using `additional_load_parameters` instead to specify update options, would they be able to?